test/documentation/seqan3.css

```diff
@@ -0,0 +1,131 @@
+body {
+    background-color: #CCC;
+    color: black;
+    font-family: -apple-system,BlinkMacSystemFont,"Segoe UI",Helvetica,Arial,sans-serif,"Apple Color Emoji","Segoe UI Emoji","Segoe UI Symbol";
+}
+
+div, p, dl {
+    font-size: 13pt;
+    font-family: -apple-system,BlinkMacSystemFont,"Segoe UI",Helvetica,Arial,sans-serif,"Apple Color Emoji","Segoe UI Emoji","Segoe UI Symbol";
+    line-height: 30px;
+}
+
+table {
+    font-size: 12pt;
+    font-family: -apple-system,BlinkMacSystemFont,"Segoe UI",Helvetica,Arial,sans-serif,"Apple Color Emoji","Segoe UI Emoji","Segoe UI Symbol";
+}
+
+div.contents {
+    margin-bottom: 10px;
+    padding: 40px;
+    margin-left: auto;
+    margin-right: auto;
+    max-width: 960px;
+    background-color: white;
+    border-radius: 8px;
+}
+
+#titlearea {
+    background-color: white;
+    align-content: center;
+}
+
+hr.footer {
+    display: none;
+}
+
+.title {
+    text-align: center;
+}
+
+.footer {
+    background-color: #AAA;
+}
+
+div.fragment {
+    font-size: 12pt;
+    line-height: 30px;
+    font-family: ubuntu-mono, sans-serif;
```

Comment:
remove this here, we don't want monospace for div

Reply:
the `div.fragment` is used for code snippets! So it is needed here.

Reply:
`<code>` is for stuff that you enclose in back ticks